### ISSUE TITLE
error: too many arguments to function ‘FILE* fopen()’ #64

### ISSUE THREAD

**Issue by ztdepztdep** [xiaoyeli/superlu_dist#116]
I copy the content of "pddrive1.c" into my c++ project, and set the include path and link with the library. the compiler fed back the following error:
error: too many arguments to function ‘FILE* fopen()’
/2T/Projects/SuperLU_dist/main.cpp:92:36: error: ‘STDC_VERSION’ was not declared in this scope

what caused this problem?

**Comment by xiaoyeli** [comment]
Can you show me Line 92 in your main.cpp code?  I suppose you are using C++ compiler?  It complains about too many arguments in fopen().

**Comment by ztdepztdep** [comment]
Yes,  I use the c++ compiler.  I want to use superlu_dist in my c++ project. how to treat this issue?

**Comment by xiaoyeli** [comment]
Can you show me Line 92 in your main.cpp code?

**Comment by ztdepztdep** [comment]
printf("__STDC_VERSION__ %ld\n", __STDC_VERSION__);


| |
Professor Ding、
|
|
***@***.***
|
---- Replied Message ----
| From | X. Sherry ***@***.***> |
| Date | 9/10/2022 14:45 |
| To | ***@***.***> |
| Cc | ***@***.***> ,
***@***.***> |
| Subject | Re: [xiaoyeli/superlu_dist] error: too many arguments to function ‘FILE* fopen()’ #64 (Issue #116) |

Can you show me Line 92 in your main.cpp code?

**Comment by xiaoyeli** [comment]
There are 2 errors:
one complains about fopen(), with too many arguments.
the other is __STDC_VERSION__ undefined. (You need to check where this is defined. )
